### ISSUE TITLE
Error When Adding a Sale Order Line

### ISSUE THREAD

**Issue by ahmed3baid** [odoo/odoo#106242]
Impacted versions: 16.0

Steps to reproduce: On developer mode, Create a quotation record and try to add a product before setting a customer.

Current behavior: Odoo Client Error

Expected behavior: Able to add a line.

Video/Screenshot link (optional):
![image](https://user-images.githubusercontent.com/23527779/203312727-5e419d9e-0cf4-40d9-9101-6142b263ac99.png)

Support ticket number submitted via odoo.com/help (optional):

**Comment by manishkumarbohra** [comment]
Hello @ahmed3baid, 

I've checked this with latest code. Unable to reproduce the issue. 

Can you pull the updated code.

**Comment by ahmed3baid** [comment]
Hi @manishkumarbohra 
Thanks for checking this.
I'm still getting this error on Runbot. It only shows on developer mode.

**Comment by manishkumarbohra** [comment]
Yes @ahmed3baid  I'm activate the developer mode and check this one. 

PFA

[soline.webm](https://user-images.githubusercontent.com/72200254/204086770-f1218825-a0eb-43eb-8e9c-fe65983d3a6b.webm)

**Comment by ahmed3baid** [comment]
@manishkumarbohra Please check the video on the link below. This seems to be working fine on CE but not EE.

https://watch.screencastify.com/v/cIcVlRrNwKGY0ISsIBwL

**Comment by werkwolk** [comment]
confirm

![image](https://user-images.githubusercontent.com/69362083/204158710-cf4410c6-e1e4-448d-af4d-f435826c664b.png)

**Comment by manuelgc1201** [comment]
Same error in EE, it trigger when drag and drop (even just try to drag) a section or note

<img width="978" alt="Screenshot 2023-01-18 at 10 02 50 PM" src="https://user-images.githubusercontent.com/73857822/213338287-6d60f7e2-d668-4dbc-967d-b85920204b79.png">

**Comment by AIOmarRehan** [comment]
@nouraellm Error has been solved, this issue should be closed, I checked it up by myself locally and through the "https://runbot.odoo.com/" too.
